src/status_im/contexts/keycard/utils.cljs

```diff
@@ -30,6 +30,13 @@
     (not has-master-key?)
     :keycard/error.keycard-empty
 
+    (and (nil? profile-key-uid)
+         (some? key-uid))
+    :keycard/error.keycard-not-empty
```

Comment:
what's the point of having this ?

Reply:
Hi @flexsurfer,

`(not= profile-key-uid key-uid)`

Wrong profile, and show different keycard (like on login)

```
    (and (nil? profile-key-uid)
         (some? key-uid))
    :keycard/error.keycard-not-empty
```

We didn't provided/or have any profile-key-uid, but keycard still returns one. Keycard is not empty. (Like while creating new account).

This was intended behavior, as we currently using wrong profile for both, but as I meantioned [above](https://github.com/status-im/status-mobile/pull/22094#discussion_r1963664912), keycard erros are not simple and we have lot of overrides.